src/apps/backend/modules/account/account-service.ts

```diff
@@ -42,6 +42,28 @@ export default class AccountService {
     return account;
   }
 
+  public static async createAccountByPhoneNumberAndName(
```

Comment:
We already have `getOrCreateAccountByPhoneNumber` and there should not be a need for this.